main_app/templates/main_app/register.html

```diff
@@ -5,60 +5,32 @@
 <body lang="en">
     {% block content %}
 
-    <body style="background-color:white; margin-top: 60px; ">
 
         <div class="register-page">
-
             <div class="container">
-                <br>
-                <div class="jumbotron" style="background-color: white;">
-                    <link href="https://fonts.googleapis.com/css2?family=Bangers&display=swap" rel="stylesheet">
-                    <h1 class="display-4" style="text-align: center; color: black; font-family: 'Bangers', cursive; ">
-                        Register</h1>
-                    <link href="https://fonts.googleapis.com/css2?family=Overpass:wght@300&display=swap"
-                        rel="stylesheet">
-                    <p class="lead" style="font-family: 'Overpass', sans-serif; text-align: center;"><b>Create Your
-                            Account</b></p>
-
-                    <!-- <img src="{% static 'Images/signup-image.jpg' %}" alt="" srcset="" style=""> -->
-                    <form method="POST" class="form-style" style="background-color: white; margin-top: 5%;   ">
-                        <img src="{% static 'Images/signup-image.jpg' %}" alt="Sign Up"
-                            style="margin-left: 32%; height: 35%; width: 35%;"><br><br>
-                        {% csrf_token %}
-                        {{form.as_p}}
-                        <br>
-                        <div style=" margin-top: 3rem; display: flex; align-items: center;
-                    justify-content: center;">
-                            <button class="btn btn-success" style=" border-radius: 25px;"
-                                type="submit"><b>Register</b></button>
-                        </div>
-                        <br>
-                        <br>
-                        <div style="display: flex; align-items: center;
-                    justify-content: center;">
-                            <strong>Already on RESCUE? </strong><a href="/login" style="color: rgb(30,142,62);"><b
-                                    class="sign-in"><u>Sign
-                                        In</u></b></a>
-                        </div>
-                    </form>
-
-                </div>
-                <br>
-                <br>
+                
```

Comment:
Make page responsive. There is a lot of space remaining for the devices with less width/

Reply:
![Register](https://user-images.githubusercontent.com/69387608/111039564-c01a4b00-8454-11eb-8386-74325b2b7e36.png)

**Fix the extra white space and make the text size bigger when viewing in mobile**

Reply:
It is because of footer and nav. I have made some changes. Please check.